docs/glossary.md

```diff
@@ -235,16 +260,17 @@ JSX
 Scoped packages
     Namespace for JavaScript packages, they provide a way to avoid naming conflicts for common package names.
 
-middleware (Redux)
-    Custom wrappers for the Redux store dispatch methods.
-    They allow customizing the behavior of the data flow inside the redux store.
+Redux
+Redux middleware
+    Custom wrappers for the [Redux](https://redux.js.org/) store dispatch methods.
+    They allow customizing the behavior of the data flow inside the Redux store.
 
-hooks (React)
-    Hooks are a React API that allow function components to use React features such as lifecycle methods, states, and so on.
+hooks
+    [Hooks](https://reactjs.org/docs/hooks-overview.html) are a React API that allow function components to use React features, such as lifecycle methods, states, and so on.
```

Comment:
There are hooks in Python too. Should be like 
- What are hooks for in prgramming
- Hooks in React
- Hooks in Python

Reply:
Rewritten in [3a96dbb](https://github.com/plone/documentation/pull/1416/commits/3a96dbbc9a2ac4d7d0f7498f76dfa3c7cab20a5d)

Reply:
I forgot to mention, there are no specific Python hooks discussed in the docs, just general programming hooks and React hooks.